generated/docs/worker-manager/schemas/v1/ruleset.json

```diff
@@ -10,10 +10,15 @@
         "type": "string"
       },
       "conditions": {
-        "type": [
-          "object",
-          "null"
-        ],
+        "type": "object",
+        "properties": {
+          "workerType": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 38,
+            "pattern": "^([a-zA-Z0-9-_]*)$"
+          }
+        },
```

Comment:
Oh, this is why it's giving you that error -- you've added a required property of conditions, and when you specify `properties` you must also specify `type`, `required` and `additionalProperties`.  However, I'm not sure this is the appropriate place to add a workerType -- @jhford would know for sure.

Reply:
Ahh alright, I'll try that.

Actually, I noticed that conditions sometimes has the workerType so I added a definition for it which is optional.

I also removed "null" from the type and instead made conditions optional by removing it from required. I believe that would have the same effect?

Reply:
I believe conditions can *optionally* contain a workerType, but can also be null.  I don't think there's a need to enforce length on that particular occurrence of workerType -- if someone has a condition with `{"workerType": "someReallyReallyReallyReallyReallyReallyReallyLongWorkerTypeName"}`, it just won't match any workerTypes since none of them have that name.

Reply:
Ahh okay I get it.  I've also added some checks for workerType occurences in conditions in some other places. Should I remove all the checks?

Reply:
I think so.  The worker-manager is still new and not yet in production, so we don't need to worry too much about it.